frontend/src/components/Calendar/DateRows.tsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react'
+import { ClickableDateContainer } from './ClickableDateContainer'
+import { EmptyDateContainer } from './EmptyDateContainer'
+
+export const dateRows = (
+  firstDate: number,
+  assignedMonth: number,
+  eventDates: number[],
+  lastDateOfThisMonth: number,
+  bufferDates: number[],
+) => {
+  for (let i = firstDate; i < lastDateOfThisMonth + 1; i++) {
+    bufferDates.push(i)
+  }
+
+  return (
+    <>
+      {bufferDates.map((day) => {
+        if (day === 0) {
+          return <EmptyDateContainer />
+        } else {
+          return (
+            <ClickableDateContainer
+              key={day}
+              date={firstDate++}
+              assignedMonth={assignedMonth}
+              eventDates={eventDates}
+            />
+          )
+        }
```

Comment:
consider tenary! may be cleaner

return (day === 0) ? <empty> : <full>

Reply:
👍🏼